cmd/nerdctl/run.go

```diff
@@ -255,6 +255,10 @@ var runCommand = &cli.Command{
 			Name:  "shm-size",
 			Usage: "Size of /dev/shm",
 		},
+		&cli.StringFlag{
+			Name:  "pid-file",
```

Comment:
Flag name should be "pidfile" for Podman compatibility.

Also could you update README.md?

Reply:
thank you @AkihiroSuda, which part should I add this  option, is it cgroup flags?

Reply:
I think this should be after "--cidfile".

Also please see "--rootfs" as an example for describing Podman-compatible flag

Reply:
it should be done @AkihiroSuda with the new commit

Reply:
Doesn't seem so

Reply:
what about now 😋

Reply:
"pid-file" still remains as of commit 2149ed874f69d8eedea7c6b243df160696feb2be

Reply:
yeah really sorry about that, it should be done 🤦🏻‍♂️